Sources/TensorFlow/Layers/Convolutional.swift

```diff
@@ -398,7 +398,7 @@ public extension Conv3D {
 /// This layer creates a convolution filter that is transpose-convolved with the layer input
 /// to produce a tensor of outputs.
 @frozen
-public struct TransposedConv2D: Layer {
+public struct TransposedConv2D<Scalar: TensorFlowFloatingPoint>: Layer {
     /// The 4-D convolution kernel.
     public var filter: Tensor<Float>
```

Comment:
Please make tensors be `Tensor<Scalar>` instead of `Tensor<Float>`. We should've noticed this earlier.

Reply:
Im surprised we didn't have any conformance here earlier,